scripts/_lines_of_code.py

```diff
@@ -55,58 +55,46 @@ class ContributedRepository:
     def __init__(self, contributed_to: Repository, user):
         self.total_lines_of_addition_in_contribution = 0
         self.total_lines_of_deletion_in_contribution = 0
-        self.excepted_lines_of_addition_in_contribution = 0
-        self.excepted_lines_of_deletion_in_contribution = 0
         self.name = contributed_to.full_name
 
-        for contribution in contributed_to.get_stats_contributors():
-            if contribution.author.id == user.id:
-                for week in contribution.weeks:
-                    self.total_lines_of_addition_in_contribution += week.a
-                    self.total_lines_of_deletion_in_contribution += week.d
-        if not any([self.total_lines_of_addition_in_contribution, self.total_lines_of_deletion_in_contribution]):
-            raise NotContributedError('User have not contributed to this repository.')
-        
+        if (contributed_to.get_commits().totalCount >= 100):
+            execute_bind('<inform>', 'This repository has a lot of commits! This can take some time...')
+
         for commit in contributed_to.get_commits():
             if commit.author.id == user.id:
                 for file in commit.files:
-                    if file.filename in excepted_files:
-                        self.excepted_lines_of_addition_in_contribution += file.additions
-                        self.excepted_lines_of_deletion_in_contribution += file.deletions
+                    if file.filename in excepted_files or file.filename.startswith('node_modules/'):
+                        pass
+                    else:
+                        self.total_lines_of_addition_in_contribution += file.additions
+                        self.total_lines_of_addition_in_contribution += file.deletions
```

Comment:
don't use the syntax:
```
if some_statement:
    pass
else:
    do_something()
```
Use this instead:
```
if not some_statement:
    do_something()
```
```suggestion
                    if not file.filename in excepted_files or file.filename.startswith('node_modules/'):
                        self.total_lines_of_addition_in_contribution += file.additions
                        self.total_lines_of_addition_in_contribution += file.deletions
```